pkg/reconciler/trigger/trigger.go

```diff
@@ -303,6 +303,27 @@ func (r *Reconciler) updateStatus(ctx context.Context, desired *v1alpha1.Trigger
 	return trig, err
 }
 
+// getBroker returns trigger's coupled broker. If the trigger is coupled with default broker, and default broker is not existed,
+// it will create the default broker first by labeling namespace with kantive-eventing-injection=enabled
+func (r *Reconciler) getBroker(ctx context.Context, t *v1alpha1.Trigger) (*v1alpha1.Broker, error) {
+	b, err := r.brokerLister.Brokers(t.Namespace).Get(t.Spec.Broker)
+	if t.Spec.Broker == "default" && apierrs.IsNotFound(err) {
+		current, _ := r.KubeClientSet.CoreV1().Namespaces().Get(t.Namespace, metav1.GetOptions{})
+		if current.Labels == nil {
+			current.Labels = map[string]string{}
+		}
+		current.Labels["knative-eventing-injection"] = "enabled"
+		_, e := r.KubeClientSet.CoreV1().Namespaces().Update(current)
+		if e != nil {
+			return b, err
```

Comment:
return e?

Reply:
I will log the error `e` here in this method. But still return `err`, which is handled here https://github.com/knative/eventing/blob/848e37d61d1121a2d0bf1fb3a38a28ab9f39f8f6/pkg/reconciler/trigger/trigger.go#L160-L168